### PR TITLE
Updating the Branding 

### PR DESCRIPTION
**Pull request by subhed**

### CODE REVIEW

**Comment by kbuffardi** [comment]
your changes are still overwriting the security patch. Please see https://github.com/kbuffardi/CodeWitUs/commit/c5595bd2232c4b6634c0084ce326abab20298c4e

**Comment by subhed** [comment]
I have now copied both the files from your commit and pasted

File: app/views/layouts/_header2.html.erb [hunk 1]
```diff
@@ -3,16 +3,46 @@
...
+                <a class="nav-link" href="/login">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Link</a>
+            </li>
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+Dropdown
+</a>
+                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
```
Comment (kbuffardi) [inline app/views/layouts/_header2.html.erb]: Most or all these changes in this file are reverting to an old version of the header, before it was corrected (to remove extraneous links and fix branding).

File: app/views/layouts/_header2.html.erb [hunk 2]
```diff
@@ -3,16 +3,46 @@
...
+                    <a class="dropdown-item" href="#">Another action</a>
+                    <div class="dropdown-divider"></div>
+                    <a class="dropdown-item" href="#">Something else here</a>
+                </div>
+            </li>
+
+
+            <!-- <li class="nav-item">
+<%# <a class="nav-link " href="/auth/google_auth2"><i class="fab fa-google"></i> Login With Google</a> %>
+        <%= button_to "Log In With Google", '/auth/google_oauth2',  {class: 'btn btn-small'}%>
```
Comment (kbuffardi) [inline app/views/layouts/_header2.html.erb]: Similarly, the security patch had changed this `button_to` to explicitly `post` but this reverts back to the older version.

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -3,15 +3,48 @@
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
 <span class="navbar-toggler-icon"></span>
 </button>
+
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
```
Comment (kbuffardi) [inline app/views/layouts/_header.html.erb]: This file has the same problems as commented in _header2.html.erb

In addition, I'm not clear why we have two headers. If they're necessary, we should give them more descriptive names. If we don't need two, we should refactor to delete one and make sure we only use the other.

**Review by kbuffardi — CHANGES_REQUESTED** [review]
See the comments for specific problems that need to be fixed.

In the future, we need to make sure to take care to always pull from `primary` and merge into the feature branch. It will also help to isolate new features by creating independent branches for different features (e.g. `authentication`, `videos`, `creator`) one at a time and then make smaller pull requests that close individual issues (along with automating closing the issue with the commit message, when appropriate e.x. `Adds Foo feature, resolves #1`)

**Comment by subhed** [comment]
Sure! Will follow that.

**Comment by kbuffardi** [comment]
Resolves #23

**Pull request merged**